docs/tutorials/substreams/getting-started.experimental.md

```diff
@@ -0,0 +1,318 @@
+---
+title: 'Getting Started'
+
+order: -9999
+outline: [0,4]
+---
+
+# Substreams Quickstart
+
+The goal of this document is to get a development environment set up in as little time as possible.
+
+Here is a quick introduction to [Substreams](https://substreams.streamingfast.io/).
```

Comment:
I think it would be better to note that the link will got outside of this document to streaming fast Substreams documentation. In fact, when I clicked the link, I was not sure where I located for a bit while.

Reply:
Change to `Substreams from StreamingFast`